### ISSUE TITLE
Codehaus link no more works

### ISSUE THREAD

**Issue by rahulsom** [jgritman/httpbuilder#42]

**Comment by reality** [comment]
Yes, need new docs on http://groovy-lang.org/ - don't think it's just moved as google search of the site doesn't turn anything up.

**Comment by melix** [comment]
Codehaus is shutting down. The documentation for Groovy itself has moved to groovy-lang.org, but third party modules like HTTPBuilder should move theirs too.

**Comment by reality** [comment]
Would it be a good idea to keep these docs hosted somewhere until they are moved?

**Comment by melix** [comment]
You can see them on http://docs.codehaus.org/display/GROOVY/Http+Builder, but be aware that Codehaus is often down, and there's nothing we can do about it, so the project maintainers should probably move the documentation to GitHub pages, for example, asap.

**Comment by reality** [comment]
This is just the summary page, the link to the project site/full documentation is 404: http://groovy.codehaus.org/modules/http-builder/

**Comment by melix** [comment]
What about generating them?

```
git clone git@github.com:jgritman/httpbuilder.git
cd httpbuilder
mvn -DskipTests site
```

then take a look at `target/site`

**Comment by reality** [comment]
Thank you, that looks good. I have mirrored the site here if anyone else is looking: http://nourishedcloud.com:81/httpbuild/

**Comment by rahulsom** [comment]
Thanks all for the links.

Would it make sense to have a CI publish docs back to the repo on the `gh-pages` branch?
A lot of grails plugins seem to be doing that.

**Comment by timyates** [comment]
I copied them to the project wiki...  Seemed a good place for them
